src/Resource.ts

```diff
@@ -21,17 +22,17 @@ export class Resource extends (BaseResource as any)
 
     public databaseName(): string
     {
-        return (this.model as any).usedConnection.options.type;
```

Comment:
this didn't work in automated tests for some reason - so I fetched that from the repository